src/csharp/Microsoft.Spark.Worker.UnitTest/TaskRunnerTests.cs

```diff
@@ -26,6 +26,13 @@ public void TestTaskRunner()
             var taskRunner = new TaskRunner(0, clientSocket, false, payloadWriter.Version);
             Task clientTask = Task.Run(() => taskRunner.Run());
 
+            TestTaskRunnerReadWrite(serverListener, payloadWriter);
+
+            Assert.True(clientTask.Wait(5000));
+        }
+
+        static internal void TestTaskRunnerReadWrite(ISocketWrapper serverListener, PayloadWriter payloadWriter)
```

Comment:
format line.  Over 100 chars.  Also `internal static` vs `static internal`

Reply:
internal static looks good to me

Reply:
resolved at 
7e715ce